manifest.json

```diff
@@ -96,6 +96,11 @@
         },
         "description": "admin/store.faviconLinks.description"
       },
+      "searchTermPath": {
+        "title": "admin/store.searchTermPath.title",
+        "type": "string",
+        "description": "admin/store.searchTermPath.description"
+      },
```

Comment:
```suggestion
```
This was removed, probably it's with an older version, so maybe a pull is necessary